README.md

```diff
@@ -54,28 +57,50 @@ Both `encode()` and `decode()` are vectorized.
 ``` r
 lorem <- unlist(lorem::ipsum(5, 1,  5))
 lorem
-#> [1] "Sit ligula senectus litora viverra consequat."          
-#> [2] "Consectetur vulputate vivamus sapien a ridiculus porta."
-#> [3] "Ipsum orci cras posuere lacus."                         
-#> [4] "Lorem nostra hendrerit nascetur vel duis consequat."    
-#> [5] "Adipiscing dui blandit vestibulum bibendum?"
+#> [1] "Dolor malesuada cursus faucibus facilisi accumsan viverra?"   
+#> [2] "Sit penatibus lobortis at aptent pellentesque!"               
+#> [3] "Sit euismod accumsan semper ante cubilia nam velit himenaeos!"
+#> [4] "Lorem pulvinar augue aliquam!"                                
+#> [5] "Dolor nullam facilisi senectus penatibus."
 
 encoded <- encode(lorem)
 encoded
-#> [1] "U2l0IGxpZ3VsYSBzZW5lY3R1cyBsaXRvcmEgdml2ZXJyYSBjb25zZXF1YXQu"                
-#> [2] "Q29uc2VjdGV0dXIgdnVscHV0YXRlIHZpdmFtdXMgc2FwaWVuIGEgcmlkaWN1bHVzIHBvcnRhLg=="
-#> [3] "SXBzdW0gb3JjaSBjcmFzIHBvc3VlcmUgbGFjdXMu"                                    
-#> [4] "TG9yZW0gbm9zdHJhIGhlbmRyZXJpdCBuYXNjZXR1ciB2ZWwgZHVpcyBjb25zZXF1YXQu"        
-#> [5] "QWRpcGlzY2luZyBkdWkgYmxhbmRpdCB2ZXN0aWJ1bHVtIGJpYmVuZHVtPw=="
+#> [1] "RG9sb3IgbWFsZXN1YWRhIGN1cnN1cyBmYXVjaWJ1cyBmYWNpbGlzaSBhY2N1bXNhbiB2aXZlcnJhPw=="    
+#> [2] "U2l0IHBlbmF0aWJ1cyBsb2JvcnRpcyBhdCBhcHRlbnQgcGVsbGVudGVzcXVlIQ=="                    
+#> [3] "U2l0IGV1aXNtb2QgYWNjdW1zYW4gc2VtcGVyIGFudGUgY3ViaWxpYSBuYW0gdmVsaXQgaGltZW5hZW9zIQ=="
+#> [4] "TG9yZW0gcHVsdmluYXIgYXVndWUgYWxpcXVhbSE="                                            
+#> [5] "RG9sb3IgbnVsbGFtIGZhY2lsaXNpIHNlbmVjdHVzIHBlbmF0aWJ1cy4="
 ```
 
 We can decode all of these using `decode()` as well. This will always
 return a `blob` object.
```

Comment:
Can we add a line here something to the effect of 

> "To achieve 0 dependencies, blob is listed only as a suggested dependency. However, the blob is always provided. If you attach the blob package, its print method will be used." 

and then `library(blob)` above the `decode(encoded)`?